src/groups/bmq/bmqp/bmqp_messageproperties.h

```diff
@@ -99,6 +99,8 @@ class MessageProperties_Schema {
                              bslma::Allocator*        basicAllocator);
     MessageProperties_Schema(const MessageProperties_Schema& other);
 
+    ~MessageProperties_Schema();
```

Comment:
Let's also add `BSLS_KEYWORD_FINAL` to the class head, since we declared a non-virtual destructor?

```
class MessageProperties_Schema BSLS_KEYWORD_FINAL {
```